components/UserTestimonies/ViewTestimony.tsx

```diff
@@ -122,16 +125,32 @@ export const FormattedTestimonyContent = ({
   const TESTIMONY_CHAR_LIMIT = 442
   const [showAllTestimony, setShowAllTestimony] = useState(false)
 
+  const _formatTestimony = (testimony: string, limit?: number) => {
+    const linkedTestimony = Autolinker.link(
+      limit ? testimony.slice(0, TESTIMONY_CHAR_LIMIT) : testimony,
+      {
+        truncate: 32
+      }
+    )
+
+    const paragraphedTestimony = linkedTestimony
+      .split(/[\r\n]+/)
+      .map((line, i) => `<p key=${i}>${line}</p>`)
+      .join("")
+
+    return parse(
```

Comment:
I think we can omit the `parse` as this is just plain html, which we can inject into the div with the [dangerouslysetinnerhtml](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) prop. So this function would return `{__html: DOMPurify...}`.

Reply:
Yeah, we can omit that, I just thought it's worse option. But I will change that :)